packages/costa/src/runtime.ts

```diff
@@ -108,6 +108,58 @@ function createRequirements(name: string, config: CondaConfig): string[] {
   return deps;
 }
 
+// TODO(Yorkie): support save caches in disk?
+/**
+ * The HTTP Cache in memory.
+ */
+interface HTTPCache {
+  etag: string;
+  lastModified: string;
+  body: any;
+}
+
+/**
+ * `HTTPCache` with the uri key.
+ */
+const httpCacheByURI: Record<string, HTTPCache> = {};
```

Comment:
Is the cache size needs to be limited for preventing excessive memory usage?

Reply:
Good catch! I added an `LRUCache` for enabling this limit, please have a double review :)